README.rst

```diff
@@ -314,13 +320,15 @@ Our online documentation:
 .. _Actions: https://github.com/parmoo/parmoo/actions
 .. _CONTRIBUTING: https://github.com/parmoo/parmoo/blob/main/CONTRIBUTING.rst
 .. _dash: https://dash.plotly.com
+.. _FAQs: https://parmoo.readthedocs.io/en/latest/faqs.html
```

Comment:
This url will be generated after merge, correct?

Reply:
yes, that will be the url of the faq section once we merge to main